src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,221 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public void add(T value) {
+        if (size == 0) {
+            first = new Node<>(null, value, null);
+            last = first;
+        } else {
+            last.next = new Node<>(last, value, null);
+            last = last.next;
+        }
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor. You should create a new `Node` with `next` and `prev` already assigned. This way, you don't have to assign `prev` and `next` to each `Node` separately.